engine/test-utils/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     implementation project(':engine-tuple')
     implementation project(':base-test-utils')
     implementation project(':engine-rowset-test-utils')
+    implementation project(':FishUtil')
```

Comment:
maybe in your next release change this from "Fish".

Reply:
Yeah, we have a cleanup issue, too. I'll see what survives.